templates/server/src/main/resources/archetype-resources/core/pom.xml

```diff
@@ -254,7 +263,19 @@
           </excludes>
         </configuration>
       </plugin>
+      
+      #if($dbMigration == 'liquibase')      
```

Comment:
can you make the indentdation consistent and also check the indendation of the generated result?